integration/testdata/bundle-list-buildpack/buildpack.toml

```diff
@@ -0,0 +1,9 @@
+api = "0.5"
+
+[buildpack]
+  id = "paketo-buildpacks/bundle-install/bundle-list"
```

Comment:
I just want to make sure I'm following - the `bundle-list` buildpack just for this test case is taking care of 2 things:
1. Replacing the need for the build plan buildpack/`plan.toml` with it's detect
2. Providing a way to list out all the installed gems?

Reply:
Yep. Exactly.